HOWTO.md

```diff
@@ -12,6 +12,7 @@ Let's present here a couple of code snippets on how to solve a couple of questio
     - [Replace all attentions from an existing ViT model with a sparse equivalent ?](#replace-all-attentions-from-an-existing-vit-model-with-a-sparse-equivalent-)
     - [Some more examples](#some-more-examples)
   - [BlockSparseAttention](#blocksparseattention)
+  - [AOTAutograd and NVFuser](#aotautograd-and-nvfuser)
```

Comment:
Maybe I would make this title more explicit? How to enable fused operations available via aotautograd/nvfuser or something like that